### DISCUSSION TITLE
Engram featured in Dygma's best layouts

### DISCUSSION THREAD

**Discussion by sunaku** [binarybottle/engram-layout#54] [General]
Engram has been featured in [Dygma's _"What's the best keyboard layout?"_ article](https://dygma.com/blogs/ergonomics/the-best-keyboard-layout) and its [accompanying YouTube video](https://www.youtube.com/watch?v=gRtS-XACO6o&t=407s), as follows:

> Developed in 2021, Engram is a project completely created from scratch, focusing mainly on the best way to type diagrams, trigrams, or... Ngrams.
>
> The punctuation in the middle creates a clear gap between the hands, which makes it ***one of the best layouts*** for split columnar keyboards.

![heatmap](https://cdn.shopify.com/s/files/1/0374/9448/9228/files/Engram_Heatmap_1024x1024.jpg?v=1677571643)

**Comment by binarybottle** [comment]
Thank you very much for the heads up, @sunaku!

**Comment by olddaniel** [comment]
I'm new to this universe, but that Dygma video (and article) helped me start my research fosusing on Colemak VS Engram. It was not very hard to choose Engram!
